src/organism.rs

```diff
@@ -53,7 +53,11 @@ impl Organism {
 
         if sensors.len() < neurons_len {
             let outputs_activations = activations.split_at(sensors.len()).1.to_vec();
+
             for n in 0..outputs.len() {
+                if n == outputs_activations.len() && outputs.len() != 1 {
```

Comment:
Apparently the  `&& outputs.len() != 1 {` part was not required. Would be nice if you accepted this pull request anyways however and changed that manually as this is my first pull request :)